robottelo/constants/__init__.py

```diff
@@ -289,9 +289,9 @@ class Colored(Box):
     'rhsc7': 'Red Hat Satellite Capsule 6.11 (for RHEL 7 Server) (RPMs)',
     'rhsc8': 'Red Hat Satellite Capsule 6.13 for RHEL 8 x86_64 (RPMs)',
     'rhsc7_iso': 'Red Hat Satellite Capsule 6.4 (for RHEL 7 Server) (ISOs)',
-    'rhsclient7': 'Red Hat Satellite Client 6 for RHEL 7 Server RPMs x86_64',
-    'rhsclient8': 'Red Hat Satellite Client 6 for RHEL 8 x86_64 RPMs',
-    'rhsclient9': 'Red Hat Satellite Client 6 for RHEL 9 x86_64 RPMs',
+    'rhsclient7': 'Red Hat Satellite Client 6 (for RHEL 7 Server) (RPMs)',
+    'rhsclient8': 'Red Hat Satellite Client 6 for RHEL 8 x86_64 (RPMs)',
+    'rhsclient9': 'Red Hat Satellite Client 6 for RHEL 9 x86_64 (RPMs)',
```

Comment:
@vsedmik we're missing this updates in master too, not sure of older .z branches, so can you update it in separate it PR please to auto-cherrypick?

Reply:
Sure, I can raise a PR to update it in `master`.